docs/new-architecture-library-intro.md

```diff
@@ -444,21 +444,19 @@ class MyComponent extends React.Component<Props> {
 
 **Creating the NativeCommands with `codegenNativeCommands`**
 
-```ts title="MyCustomMapNativeComponent.js"
-import codegeNativeCommands from 'react-native/Libraries/Utilities/codegenNativeCommands';
-...
-type Props = {...};
+```js title="MyCustomMapNativeComponent.js"
```

Comment:
Let's keep `ts` here. It's not TypeScript but Flow, and this gives us the better highlightning